_dprhtml/js/send.js

```diff
@@ -4,6 +4,7 @@ var DPR1_send_mod = ( function () {
 
 function eventSend(event,internal) {
   if(!event) return;
+  if(internal == 1) return 'search';

```

Comment:
What else is the internal flag used for? See down below it is also checked. Looks like we are overloading the meaning of internal. This function is scary because it is used in nearly all event handlers and has multiple copies

Reply:
Yes, I overloaded internal to avoid a third parameter. Except for this case, the value of internal is never an integer.